sdk/communication/azure-communication-sms/azure/communication/sms/_shared/models.py

```diff
@@ -187,15 +186,14 @@ def _microsoft_teams_user_raw_id(identifier):
     return '8:orgid:{}'.format(user_id)
 
 
-def identifier_from_raw_id(raw_id):
+def identifier_from_raw_id(raw_id: str) -> CommunicationIdentifier:
     """
     Creates a CommunicationIdentifier from a given raw ID.
 
     When storing raw IDs use this function to restore the identifier that was encoded in the raw ID.
 
     :param raw ID to construct the CommunicationIdentifier from.
```

Comment:
```suggestion
    :param str raw_id: A raw ID to construct the CommunicationIdentifier from
```
we should keep the reST docstring format, as it's used everywhere in this repo. Same suggestion for other occurrences in other acs modalities